utils/service_discovery/sd_docker_backend.py

```diff
@@ -290,6 +301,12 @@ def get_tags(self, state, c_id):
             if swarm_svc:
                 tags.append('swarm_service:%s' % swarm_svc)
 
+        if Platform.is_rancher():
+            c_inspect = state.inspect_container(c_id)
+            service_name = c_inspect.get('Config', {}).get('Labels', {}).get('io.rancher.container.name')
```

Comment:
maybe make `io.rancher.container.name` a constant? It could live here or in dockerutil.